velox/common/memory/Memory.h

```diff
@@ -225,7 +225,7 @@ class MemoryManager {
 
   /// Invoked to shrink alive pools to free 'targetBytes' capacity. The function
   /// returns the actual freed memory capacity in bytes.
-  uint64_t shrinkPools(uint64_t targetBytes);
+  uint64_t shrinkPools(MemoryPool* requestor, uint64_t targetBytes);
```

Comment:
Why need requestor? We could use this API for server memory pushback later in Prestissimo?
Also shall we add a 'fast'/'slow' or 'canSpill' to either reclaim memory by killing a query or spill? Thanks!